### PR TITLE
Aarosh new branchlol

### PR DESCRIPTION
**Pull request by LilMarz**
What do you think about this??? Also, decimals don't work. Have a nice day bye bye have a great time!!!

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -1 +1,17 @@
-#It will calculate few calculations lol
+#I like to steal your cookies then dip them in the soda dispenser
+#Imagine being a bunyan plus going to a school with 21 kids plus having no arm nor leg hair and not being in summa
+#Imagine not having leg hair and having only indian friends and being hot
+#Imagine not being in summa and using discord during school and being a purid bacha
+
+def addition():
+    num = int(input("Give me number pls"))
```
Comment (IanRBr) [inline calculator.py:8]: Dude you told me to be serious this isn't being serious why would you say, "Give me another number bruh bing chilling sussy boy" you know my dad is going to see this right? Also why the hell are you calling me hot

File: calculator.py [hunk 2]
```diff
@@ -1 +1,17 @@
...
+    num2 = int(input("Give me another number bruh bing chilling sussy boy"))
+    numans = num + num2
+    return numans
```
Comment (Oozegy) [inline calculator.py:11]: it sucks

**Review by AosafRahman — COMMENTED** [review]
its log

**Review by Oozegy — APPROVED** [review]

**Review by AosafRahman — APPROVED** [review]
it joggers

**Review by IanRBr — APPROVED** [review]
this is alright

**Pull request merged**